src/main/java/appeng/entity/AEBaseEntityItem.java

```diff
@@ -40,8 +40,7 @@ public AEBaseEntityItem( final World world, final double x, final double y, fina
 		super( world, x, y, z, stack );
 	}
 
-	@SuppressWarnings( "unchecked" )
-	public List<Entity> getCheckedEntitiesWithinAABBExcludingEntity( final AxisAlignedBB region )
+	@SuppressWarnings( "unchecked" ) List<Entity> getCheckedEntitiesWithinAABBExcludingEntity( final AxisAlignedBB region )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Annotation 'SuppressWarnings' should be alone on line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.annotation.AnnotationLocationCheck)